### ISSUE TITLE
Random Crash with Server Finder

### ISSUE THREAD

**Issue by scriptinator** [Wurst-Imperium/Wurst-Client#993]

**Comment by WurstContributer** [comment]
When including error output, please wrap it into a Markdown code block so that it renders nicely.
Example:

```
java.lang.NullPointerException: null
    at org.apache.logging.log4j.core.config.AppenderControl.callAppender(AppenderControl.java:89)
    at org.apache.logging.log4j.core.config.LoggerConfig.callAppenders(LoggerConfig.java:425)
    at org.apache.logging.log4j.core.config.LoggerConfig.log(LoggerConfig.java:406)
    at org.apache.logging.log4j.core.config.LoggerConfig.log(LoggerConfig.java:367)
    at org.apache.logging.log4j.core.Logger.log(Logger.java:110)
```

**Comment by WurstContributer** [comment]
Hard to read
Please put Three of ` <---
Around the text

**Comment by mcccs** [comment]
fixed verison
```
*---------------- Wurst Client Crash Report ----------------
---- Please report this crash at: wurst-client.tk/bugs ----
// Surprise! Haha. Well, this is awkward.

Time: 6/19/15 9:25 AM
Description: Rendering screen

java.lang.IndexOutOfBoundsException: Index: 22, Size: 0
at java.util.ArrayList.rangeCheck(ArrayList.java:653)
at java.util.ArrayList.get(ArrayList.java:429)
at net.minecraft.client.gui.ServerSelectionList.getListEntry(ServerSelectionList.java:45)
at net.minecraft.client.gui.GuiListExtended.drawSlot(GuiListExtended.java:31)
at net.minecraft.client.gui.GuiSlot.drawSelectionBox(GuiSlot.java:483)
at net.minecraft.client.gui.GuiSlot.drawScreen(GuiSlot.java:260)
at net.minecraft.client.gui.GuiMultiplayer.drawScreen(GuiMultiplayer.java:502)
at net.minecraft.client.renderer.EntityRenderer.updateCameraAndRender(EntityRenderer.java:1198)
at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1327)
at net.minecraft.client.Minecraft.run(Minecraft.java:412)
at net.minecraft.client.main.Main.main(Main.java:127)

A detailed walkthrough of the error, its code path and all known details is as follows:

-- Head --
Stacktrace:
at java.util.ArrayList.rangeCheck(ArrayList.java:653)
at java.util.ArrayList.get(ArrayList.java:429)
at net.minecraft.client.gui.ServerSelectionList.getListEntry(ServerSelectionList.java:45)
at net.minecraft.client.gui.GuiListExtended.drawSlot(GuiListExtended.java:31)
at net.minecraft.client.gui.GuiSlot.drawSelectionBox(GuiSlot.java:483)
at net.minecraft.client.gui.GuiSlot.drawScreen(GuiSlot.java:260)
at net.minecraft.client.gui.GuiMultiplayer.drawScreen(GuiMultiplayer.java:502)

-- Screen render details --
Details:
Screen name: net.minecraft.client.gui.GuiMultiplayer
Mouse location: Scaled: (278, 219). Absolute: (834, 242)
Screen size: Scaled: (480, 300). Absolute: (1440, 900). Scale factor of 3
Stacktrace:
at net.minecraft.client.renderer.EntityRenderer.updateCameraAndRender(EntityRenderer.java:1198)
at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1327)
at net.minecraft.client.Minecraft.run(Minecraft.java:412)
at net.minecraft.client.main.Main.main(Main.java:127)

-- System Details --
Details:
Minecraft Version: 1.8
Wurst Client Version: 1.15 (latest: 1.15)
Operating System: Mac OS X (x86_64) version 10.10.2
Java Version: 1.8.0_25, Oracle Corporation
Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
Memory: 350865928 bytes (334 MB) / 686514176 bytes (654 MB) up to 1060372480 bytes (1011 MB)
JVM Flags: 5 total; -Xmx1G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
IntCache: cache: 0, tcache: 0, allocated: 13, tallocated: 95
Launched Version: Wurst
LWJGL: 2.9.2
OpenGL: Intel HD Graphics 5000 OpenGL Engine GL version 2.1 INTEL-10.2.46, Intel Inc.
GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because ARB_framebuffer_object is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.*

Using VBOs: No
Is Modded: Very likely; Jar signature invalidated
Type: Client (map_client.txt)
Resource Packs: [faithful32pack, faithful64pack-v1.4.0]
Current Language: English (US)
Profiler Position: N/A (disabled)
```

**Comment by Alexander01998** [comment]
Duplicate: #848
